core/proto/keytransparency_v1_types/keytransparency_v1_types.proto

```diff
@@ -64,19 +65,13 @@ message Entry {
   bytes previous = 3;
 }
 
-// KeyValue is a map entry.
-message KeyValue {
-  // key contains the map entry key.
-  bytes key = 1;
-  // TODO(ismail): make proto.Any
-  // value contains the map entry value.
-  bytes value = 2;
-}
-
 // SignedKV is a signed change to a map entry.
 message SignedKV {
-  // key_value is a serialized KeyValue.
-  KeyValue key_value = 1;
+  // index is the location of this leaf in the sparse merkle tree.
```

Comment:
You should probably make it clear that tag 1 is not to be re-used by adding

reserved 1;  // key_value